carma-messenger-core/emergency_response_vehicle_plugin/include/emergency_response_vehicle_plugin/emergency_response_vehicle_plugin_node.hpp

```diff
@@ -168,24 +161,10 @@ namespace emergency_response_vehicle_plugin
      */
     void publishBSM();
 
-    /**
-     * \brief Service callback for a request of the name of the final destination of an Emergency Response Vehicle's 
-     * planned route. This service is called by the Emergency Response Web UI Widget at initial startup of the UI.
-     * \param req An empty carma_planning_msgs::srv::GetEmergencyRoute::Request.
-     * \param resp A carma_planning_msgs::srv::GetEmergencyRoute::Response with fields populated to indicate whether the route
-     * file provided in config_.emergency_route_file_path was properly processed by this node, and to provide the name of the
-     * Emergency Resonse Vehicle's route.
-     */
-    void getEmergencyRouteCallback(
-      std::shared_ptr<rmw_request_id_t>, 
-      carma_planning_msgs::srv::GetEmergencyRoute::Request::SharedPtr req, 
-      carma_planning_msgs::srv::GetEmergencyRoute::Response::SharedPtr resp);
-
     /**
      * \brief Service callback for a service call that indicates the Emergency Response Vehicle has arrived at its destination.
      * Service is called by the Emergency Response Web UI Widget, and results in this plugin clearing its route_destination_points_
-     * so that future route destination points will no be included in the broadcasted BSMs. Additionally, route_final_destination_name_
-     * is cleared as well.
+     * so that future route destination points will no be included in the broadcasted BSMs.
```

Comment:
Isn't the future route destination points in the regional extension of BSM? also `no` to `not`.